adoptingerlang.org

```diff
@@ -1836,7 +1836,7 @@ The first =relx= step seen in the output is "Resolving OTP Applications" followe
 #+attr_shortcode: info
 #+attr_latex: :environment bclogo :options [logo=\bcinfo, couleurBarre=orange, noborder=true, couleur=white]{Information}
 #+begin_notice
-Note that the application <code>sasl</code> is not included in the <code>relx</code> config's list of applications. By default in the rebar3 template it is in the list. This is because <code>sasl</code> is required for certain release operations. The <code>sasl</code> application includes <code>release_handler</code> which provides functionality for performing release upgrades and downgrades. Since we are focused here on creating containers which are replaced instead of live upgraded <code>sasl</code> does not need to be included.
+Note that the application <code>sasl</code> is not included in the <code>relx</code> config's list of applications. By default in the rebar3 template it is in the list. This is because <code>sasl</code> is required for certain release operations. The <code>sasl</code> application includes <code>release&#95;handler</code> which provides functionality for performing release upgrades and downgrades. Since we are focused here on creating containers which are replaced instead of live upgraded <code>sasl</code> does not need to be included.
```

Comment:
curious about what's up with the &#95; entity here, which afaict is just the underscore. I guess some parsing blew up otherwise?

Reply:
Without doing it this way the `_` ends up being `\_` in the result.